src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -320,11 +321,19 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
 
                     if ((object)overriddenMethod != null)
                     {
-                        CustomModifierUtils.CopyMethodCustomModifiers(overriddenMethod, this, out _lazyReturnType, 
-                                                                      out _lazyCustomModifiers, 
+                        CustomModifierUtils.CopyMethodCustomModifiers(overriddenMethod, this, out _lazyReturnType,
+                                                                      out _lazyCustomModifiers,
                                                                       out _lazyParameters, alsoCopyParamsModifier: true);
                     }
                 }
+                else if (_refKind == RefKind.RefReadOnly)
```

Comment:
Filed #20053 to track that.

---
In reply to: [119768491](https://github.com/dotnet/roslyn/pull/19658#discussion_r119768491) [](ancestors = 119768491)